model/account/accounts.go

```diff
@@ -331,8 +332,17 @@ func init() {
 
 			createSoftDeletedAccount(db, old)
 
-			_, err = PushAccountDeletedJob(jobsSystem, db, old.ID(), old.Rev(), konnector)
-			return err
+			// Execute the OnDeleteAccount if the konnector has declared one
+			man, err := app.GetKonnectorBySlug(db, konnector)
+			if err != nil {
```

Comment:
Could we have a case where we delete an account for a konnector that has already been uninstalled? I don't see one, but I'm not confident on that.